modules/developer-cli-odo-installing-odo-on-vs-code.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/developer_cli_odo/installing-odo.adoc
+
+[id="installing-odo-on-vs-code"]
+
+= Installing odo on VS Code
+
+The https://marketplace.visualstudio.com/items?itemName=redhat.vscode-openshift-connector[OpenShift VS Code extension] uses both `odo` and the `oc` binary to interact with your Kubernetes or OpenShift cluster. Install odo on VS Code to work with these features.
```

Comment:
Suggest removing "Kubernetes or". Because these are specifically OpenShift docs.

Actually I would also replace "OpenShift" with {product-title}, because I think these docs show for OKD. So:

```
... to interact with your {product-title} cluster.
```

Reply:
> Install odo on VS Code to work with these features.

Are you installing odo on VS code with this procedure, or should you say that you are installing this OpenShift VS Code extension on VS Code?